Topics FAQ update (#2589)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/faq/index.md

Topics are only inferred from the hostname and not from the URL path.

> ### Can I control which third parties on my page are allowed to infer topics from my users?

Reviewer: ```suggestion
### Can I control which third parties have access to topics on my page?
```

Reviewer: @antoinebisch I want to confirm this doesn't change the meaning. Otherwise looks good

> Yes. You can control access to the API for third party domains on your page with the Permission Policy header.

Reviewer: ```suggestion
Yes. You can use the Permission Policy header to control third-party access to the Topics API on your page.
```

Use `self` and any domains you would like to allow access to the API as parameters.

> As an example: to completely disable use of the Topics API within all browsing contexts except for your own origin and those whose origin is "https://example.com", set the following HTTP response header: `Permissions-Policy: geolocation=(self "https://example.com")`.

Reviewer: ```suggestion
For example, to completely disable use of the Topics API within all browsing contexts except for your own origin and those whose origin is `https://example.com`, set the following HTTP response header: 'Permissions-Policy: geolocation=(self "https://example.com")`
```
